codingGuideLines/cpp.md

```diff
@@ -484,17 +476,25 @@ struct BobClass
 ```
 
 
-## 17. Template Class / Struct and Type Definitions
-  - avoid the keyword `class` inside template definitions
+## 16. Template Declarations and Specializations
+  - type ( class, struct ) template parameters shall be prefixed with T_ followed by an **upper case letter** camel case
```

Comment:
please add syntax highlighting to T_ in the form of `T_` (use the small markdown "ticks" `)